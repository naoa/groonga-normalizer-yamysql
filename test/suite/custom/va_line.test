register normalizers/yamysql

normalize NormalizerYaMySQL "ヴァヴィヴヴェヴォ"
