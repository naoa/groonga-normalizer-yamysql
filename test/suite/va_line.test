register normalizers/yamysql

normalize NormalizerYaMySQLUnicodeCI "ヴァヴィヴヴェヴォ"
