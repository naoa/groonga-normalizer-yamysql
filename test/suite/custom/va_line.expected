register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQL "ヴァヴィヴヴェヴォ"
[[0,0.0,0.0],{"normalized":"バビヴベボ","types":[],"checks":[]}]
